src/snowflake/connector/result_batch.py

```diff
@@ -96,9 +92,7 @@ def col_to_converter(col: dict[str, Any]) -> tuple[str, SnowflakeConverterType]:
             )
             return type_name, python_method
 
-        column_converters: list[tuple[str, SnowflakeConverterType]] = [
-            col_to_converter(c) for c in rowtypes
-        ]
+        column_converters = [col_to_converter(c) for c in rowtypes]
```

Comment:
emmm, we're removing type hint here?

Reply:
Yes, believe it or not this fixed an error. TBH I cannot remember if the type-hint was incorrect here, or if `mypy` can just infer the actual type and didn't want to be told what this was.